pkg/controllers/provisioning/v1alpha1/allocation/suite_test.go

```diff
@@ -87,15 +87,8 @@ var _ = Describe("Allocation", func() {
 
 	Context("Reconcilation", func() {
 		It("should provision nodes for unconstrained pods", func() {
-			pods := []*v1.Pod{
-				test.PodWith(test.PodOptions{
-					Conditions: []v1.PodCondition{{Type: v1.PodScheduled, Reason: v1.PodReasonUnschedulable, Status: v1.ConditionFalse}},
-				}),
-				test.PodWith(test.PodOptions{
-					Conditions: []v1.PodCondition{{Type: v1.PodScheduled, Reason: v1.PodReasonUnschedulable, Status: v1.ConditionFalse}},
-				}),
-			}
-			for _, pod := range pods {
+			ps := []*v1.Pod{test.PendingPod(), test.PendingPod()}
```

Comment:
you can keep this variable name as pods now, if you choose.

Reply:
D'oh!!!!!